Neo4j.Driver/Neo4j.Driver/Internal/Routing/LoadBalancer.cs

```diff
@@ -128,22 +126,28 @@ public Task CloseAsync()
 
             return Task.CompletedTask;
         }
-
-        public async Task VerifyConnectivityAsync()
+        
+        public async Task<IServerInfo> VerifyConnectivityAndGetInfoAsync()
         {
-            // As long as there is a fresh routing table, we consider we can route to these servers.
-            try
+            var supportsMultiDb = await SupportsMultiDbAsync().ConfigureAwait(false);
```

Comment:
Would this be better inside the try/catch block? Or do we want any potential errors from this to propagate as them selves rather than being wrapped in the VerifyConnectivity errors?